pymongo/encryption.py

```diff
@@ -552,6 +554,95 @@ def __init__(
         # Use the same key vault collection as the callback.
         self._key_vault_coll = self._io_callbacks.key_vault_coll
 
+    def create_encrypted_collection(
+        self,
+        database: Database,
+        name: str,
+        kms_provider: Optional[str] = None,
+        data_key_opts: Optional[Mapping[str, Any]] = None,
+        **kwargs: Any,
+    ) -> Tuple[Collection[_DocumentType], Mapping[str, Any]]:
+        """Create a collection with encryptedFields.
+
+        Normally collection creation is automatic. This method should
+        only be used to specify options on
+        creation. :class:`~pymongo.errors.CollectionInvalid` will be
+        raised if the collection already exists.
+
+        :Parameters:
+          - `name`: the name of the collection to create
+          - `kms_provider`: the KMS provider to be used
+          - `data_key_opts` (dict): a dictionary containing additional arguments to the `create_data_key` method that will be
+            used to auto-generate the data keys if `keyId` is `None` such as::
+
+                {
+                   masterKey: Optional<Document>
+                   keyAltNames: Optional<Array[String]>
+                   keyMaterial: Optional<BinData>
+                }
```

Comment:
These params are actually snake_case not camelCase. What do you think about exposing these explicitly instead of via a dict? Like:
```
    def create_encrypted_collection(
        self,
        database: Database,
        name: str,
        kms_provider: Optional[str] = None,
        master_key: Optional[Mapping[str, Any]] = None,
        key_alt_names: Optional[Sequence[str]] = None,
        key_material: Optional[bytes] = None,
```

Reply:
Done.